src/config.js

```diff
@@ -6,6 +6,10 @@ export const localStorageName = 'KedroViz';
 // These values are used in both SCSS and JS, and we don't have variable-sharing
 // across Sass and JavaScript, so they're defined in two places. If you update their
 // value here, please also update their corresponding value in src/styles/_variables.scss
+export const metaSidebarWidth = {
+  open: 400,
+  closed: 0
+};
```

Comment:
It looks like the CSS is just using the `sidebarWidth` values. If it's worth adding different ones for the JS, to be consistent you should use different variables for the CSS too.

Reply:
Good point, I've added the new constants in CSS too.